Services/Administration/OJS.Services.Administration.Business/Contests/ContestsBusinessService.cs

```diff
@@ -119,7 +119,11 @@ public override async Task<ContestAdministrationModel> Create(ContestAdministrat
     {
         var contest = model.Map<Contest>();
 
-        AddProblemGroupsToContest(contest, model.NumberOfProblemGroups);
+        if (model.IsOnlineExam && model.NumberOfProblemGroups == 0)
+        {
+            AddProblemGroupsToContest(contest, model.NumberOfProblemGroups);
+        }
```

Comment:
Why is this check? I don't get it, why `== 0`?

Reply:
Fixed.